web/src/lib/roadmap/buildUserRoadmap.test.ts

```diff
@@ -866,4 +866,29 @@ describe("buildUserRoadmap", () => {
       expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).toEqual([]);
     });
   });
+
+  describe("raffle-bingo games business", () => {
+    it("adds raffle-bingo-games task if raffle-bingo-games business", () => {
+      const profileData = generateStartingProfile({
+        raffleBingoGames: true,
+        isNonprofitOnboardingRadio: true,
```

Comment:
This line isn't impacting the logic in `buildUserRoadmap`. Strongly advise we leave this out. This is a case where we're staging data that does not actually support the test case, which is something else we've been talking about as a team.
```suggestion
```